src/Mono.Android/metadata

```diff
@@ -1922,6 +1922,10 @@
   <attr api-since="35" path="/api/package[@name='android.media']/class[@name='MediaRouter2']/method[@name='getRouteListingPreference' and count(parameter)=0]" name="propertyName"></attr>
   <attr api-since="35" path="/api/package[@name='android.media']/class[@name='MediaRouter2']/method[@name='setRouteListingPreference' and count(parameter)=1 and parameter[1][@type='android.media.RouteListingPreference']]" name="propertyName"></attr>
 
+  <!-- These new default interface methods on a Listener interface cause duplicate named EventHandlers to be generated, change to a unique name. -->
```

Comment:
Wouldn't it be better to add `eventName` metadata to explicitly name the event, vs. renaming the method name itself?

Reply:
Unfortunately, it isn't the event itself that is the problem[1], it's the `EventHandler` name in the `internal` `*ListenerImplementor` that is the problem:

```csharp
[global::Android.Runtime.Register ("mono/android/app/AppOpsManager_OnOpActiveChangedListenerImplementor", ApiSince = 30)]
internal sealed partial class IOnOpActiveChangedListenerImplementor : global::Java.Lang.Object, IOnOpActiveChangedListener
{
    public EventHandler<OpActiveChangedEventArgs>? OnOpActiveChangedHandler;

    public void OnOpActiveChanged (string op, int uid, string packageName, bool active)
    {
    	var __h = OnOpActiveChangedHandler;
    	if (__h != null)
    		__h (sender, new OpActiveChangedEventArgs (op, uid, packageName, active));
    }

    // Duplicate OnOpActiveChangedHandler
    public EventHandler<OpActiveChangedEventArgs>? OnOpActiveChangedHandler;

    public void OnOpActiveChanged (string op, int uid, string packageName, string? attributionTag, int virtualDeviceId, bool active, int attributionFlags, int attributionChainId)
    {
    	var __h = OnOpActiveChangedHandler;
    	if (__h != null)
    		__h (sender, new OpActiveChangedEventArgs (op, uid, packageName, attributionTag, virtualDeviceId, active, attributionFlags, attributionChainId));
    }
}
```

[1] I can't find anything that actually generates events for this listener (ie: no `SetOnOpActiveChangedListener` method anywhere).  If that existed then the event name would likely _also_ be a problem.

Reply:
Previously, we have also just removed these new methods, as they are `default` interface methods, so we could also do that here as well:

https://github.com/xamarin/xamarin-android/blob/ef31401a44311ed49cee9df1e207918226b4c745/src/Mono.Android/metadata#L1701-L1703